packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -272,6 +286,81 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
         isRegistered.set(false)
     }
 
+    @Suppress("TooGenericExceptionCaught")
+    private fun mapLogEvent(event: LogEvent): LogEvent? {
```

Comment:
is it possible to add tests for this logic? or it will be too complicated? I can help if needed.

Reply:
I'll give it a shot. The complexity of the different dependencies makes it difficult which is why I focused on the integration test.